packages/landing/index.html

```diff
@@ -138,7 +139,7 @@ <h4 class="text-lg font-bold mb-3">Text Critical Notes</h4>
       </section>
       <section
         id="volunteer"
-        class="bg-blue-800 pt-32 pb-16 md:pb-24 lg:pb-32 px-6 md:px-8"
+        class="pt-12 pb-16 md:pb-24 lg:pb-32 px-6 md:px-8"
```

Comment:
I moved 80px of top padding from this item to the bottom padding of the above section.